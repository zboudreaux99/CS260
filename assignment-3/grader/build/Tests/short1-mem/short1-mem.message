Output line 0:
  <i>expected: </i>shortList1.txt contained <span class='expected'>190</span> books. shortList2.txt contained 190 books. Their merge contains
  <i>observed: </i>shortList1.txt contained <span class='observed'>0</span> books. shortList2.txt contained 0 books. Their merge contains 0
