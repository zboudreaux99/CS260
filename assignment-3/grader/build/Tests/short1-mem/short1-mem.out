shortList1.txt contained 0 books.
shortList2.txt contained 0 books.
Their merge contains 0 books.

--- end of output ---
--- std error ---
Memory leak detected: 600
